### PR TITLE
News API rfc

### PR DESCRIPTION
**Pull request by hitesh22rana**

### CODE REVIEW

**Comment by hitesh22rana** [comment]
@2002Bishwajeet rfc for news api and package is added

File: 0003-news-api.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+[design-proposal]: #design-proposal
+
+We will be dividing the proposal into Five phases :-
+
+### First Phase
+This will be the inital phase , we will try to gather all the website as much as we can with different types of news coverage, so that we would have a clear cut idea of what website to scrape data from
+
+### Second Phase
+In this phase , we will be making different classes based approach for various categories of news to scarpe data.
```
Comment (2002Bishwajeet) [inline 0003-news-api.md]: scrape

File: 0003-news-api.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### Third Phase
+In this phase we will optimize our code for better and fast scraping as well as to hanlde exceptions to inimize crashes , and we will also deploy our package in PyPy official library of python.
```
Comment (2002Bishwajeet) [inline 0003-news-api.md]: minimize

Comment (2002Bishwajeet) [reply 0003-news-api.md]: Handle

**Review by 2002Bishwajeet — COMMENTED** [review]
Looks good just fix some minor spelling mistakes

**Pull request merged**